dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -1132,6 +1160,151 @@ public void shouldReturnParserContainerUUID() throws Exception {
         Assert.assertTrue(html.matches(regexExpected));
     }
 
+    /**
+     * Method to test: {@link com.dotmarketing.portlets.htmlpageasset.business.render.HTMLPageAssetRenderedAPIImpl#getPageHtml(PageContext, HttpServletRequest, HttpServletResponse)}
+     * Given Scenario: Create a page with File Container liked with relative path in the template
+     * ExpectedResult: should work
+     *
+     * @throws Exception
+     */
+    @Test
+    public void shouldRenderRelativeContainerPath() throws Exception {
+
+        final FileAssetContainer container = createFileContainer();
+        final Template template = new TemplateDataGen().title("PageContextBuilderTemplate"+System.currentTimeMillis())
+                .host(site)
+                .withContainer(container.getPath(), UUID)
+                .nextPersisted();
+        PublishFactory.publishAsset(template, systemUser, false, false);
+
+        final String pageName = "testPage-"+System.currentTimeMillis();
+        final HTMLPageAsset pageEnglishVersion = createHtmlPageAsset(template, pageName, 1);
+
+        createMultiTree(pageEnglishVersion.getIdentifier(), container.getIdentifier());
+
+        final HttpServletRequest mockRequest = createHttpServletRequest(pageEnglishVersion);
+        Mockito.when(mockRequest.getParameter(WebKeys.PAGE_MODE_PARAMETER)).thenReturn(PageMode.LIVE.toString());
+
+        final HttpServletResponse mockResponse = mock(HttpServletResponse.class);
+        final HttpSession session = createHttpSession(mockRequest);
+        Mockito.when(session.getAttribute(WebKeys.VISITOR)).thenReturn(null);
+        Mockito.when(session.getAttribute(WebKeys.CMS_USER)).thenReturn(systemUser);
+
+        final String html = APILocator.getHTMLPageAssetRenderedAPI().getPageHtml(
+                PageContextBuilder.builder()
+                        .setUser(systemUser)
+                        .setPageUri(pageEnglishVersion.getURI())
+                        .setPageMode(PageMode.LIVE)
+                        .build(),
+                mockRequest, mockResponse);
+
+        Assert.assertEquals("content2content1", html);
+    }
+
+    /**
+     * Method to test: {@link com.dotmarketing.portlets.htmlpageasset.business.render.HTMLPageAssetRenderedAPIImpl#getPageHtml(PageContext, HttpServletRequest, HttpServletResponse)}
+     * Given Scenario: Create a page and a advance template using a File Container in another site
+     * ExpectedResult: should work
+     *
+     * @throws Exception
+     */
+    @Test
+    public void shouldRenderUsingOtherSiteContainer() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5067852)